### ISSUE TITLE
"Add Item" Buttons are not clickable via the keyboard

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1793]
### Description

In Kanban, when I navigate using the keyboard and reach the buttons for adding new items to a board, I’m unable to click on them.

![Image](https://github.com/user-attachments/assets/4f70e21c-bc75-4edf-9084-be0132bf6b6e)

### Steps to reproduce

1. Open Kanban and navigate using only the keyboard.  
2. While focused on one of the buttons for adding a new item, try pressing Enter.

### Relevant logs

```shell

```

### Expected behavior

The buttons should work when using the keyboard also

### Which operating system are you using?

Microsoft Windows

### What browsers are you seeing the problem on?

Mozilla Firefox

### Extensions

.

### Version

2024.12.0
